new/LIT6959Laemafatari.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6959Laemafatari" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለእመ፡ ፈጣሪ፡ ድንግል፡ እንተ፡ እምፍጡራን፡ ሐረያ፤</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-ʾǝmma faṭāri dǝngǝl ʾǝnta ʾǝm-fǝṭurān ḥarayā, ...  (Qǝne of ʿǝṭāna mogar type)</title>
+                <title xml:lang="en" corresp="#t1">He chose the mother of the Creator, the Virgin, among creatures, ...</title>
```

Comment:
(Spontaneously, I would say "If the creator chose a virgin, who is from among the creatures, ...")

Reply:
I think it is not. You should understand it rather like: ለእመ፡ ፈጣሪ፡ ድንግል፡.... ዜና፡ ዕበያ፡ ይነግር፡ መጽሐፈ፡ ተአምር፡ According to your proposal, it would be perhaps ለእመ፡ ሐረያ. But perhaps multiple interpreration is also possible here.